pkg/util/cmapichecker/cmapichecker.go

```diff
@@ -32,10 +31,10 @@ import (
 )
 
 var (
-	ErrCertManagerCRDsNotFound   = errors.New("the cert-manager CRDs are not yet installed on the Kubernetes API server")
```

Comment:
This removes unnecessary traceback info from these errors because this traceback info renders quite poorly in json logs.